### PR TITLE
feat: add mentors section

### PR DESCRIPTION
**Pull request by ashutosh-rath02**
Added typography for smaller screens

### CODE REVIEW

**Comment by ashutosh-rath02** [comment]
@Shurtu-gal Can you please review this?

File: src/components/atoms/MentorCard/style.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  align-items: center;
+  width: 500px;
+  height: 480px;
+  // &:hover {
+  //   animation: ${scaleAnimation} 0.3s ease-in-out forwards;
+  // }
+  // animation: ${scaleDownAnimation} 0.3s ease-in-out forwards;
+  border-radius: 10px;
+
+  img {
```
Comment (Shurtu-gal) [inline src/components/atoms/MentorCard/style.js:13]: image
The aspect ratio is not being maintained, please use the proper object-fit.

Comment (Shurtu-gal) [reply src/components/atoms/MentorCard/style.js:13]: Also border-radius should be more

**Review by Shurtu-gal — COMMENTED** [review]
Also, change the title 😃

**Comment by ashutosh-rath02** [comment]
On it

File: src/components/atoms/MentorCard/style.js (hunk 1 above)
Comment (ashutosh-rath02) [reply src/components/atoms/MentorCard/style.js:13]: This slider needs to cover entire width.

File: config/content/Mentors.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+export default {
```
Comment (Shurtu-gal) [inline config/content/Mentors.js]: @Lucif3r-in Put this in data folder.

**Review by Shurtu-gal — APPROVED** [review]
LGTM 🚀

**Pull request merged**